docs/repositories-configure/managing-branches.md

```diff
@@ -1,12 +1,36 @@
 # Managing branches
 
+{% if not extra.self_hosted %}
+Codacy automatically analyzes the default branch of your repository (typically `master` or `main` as configured on your Git provider), and also supports analyzing multiple branches.
+
+To change the default branch of your repository or enable analysis on other branches:
+
+1.  Open your repository **Settings**, tab **Branches**.
+
+    ![Managing branches](images/managing-branches.png)
+
+1.  To enable a new branch, toggle the corresponding switch in the column **Analyze**.
+
+    Enabling a new branch triggers an initial analysis of that branch and the analysis results and information for that branch will become available after the analysis is complete.
+
+1.  To change the default branch on Codacy, click the corresponding **Make default** link in the column **Default**.
+
+    Codacy displays information for the default branch on dashboards and the repositories list by default, and we recommend that you set the default branch on Codacy to match the default branch of your Git repository.
+
+    !!! note
+        You can only set as the default branch an already analyzed branch.
+{% else %}
 Codacy automatically triggers analysis on the main branch of your repository (typically `master` or `main` as configured on your Git provider), and also supports analyzing multiple branches.
 
 To change the main branch of your repository or enable analysis on other branches, open your repository **Settings**, tab **Branches**. Enabling a new branch triggers an initial analysis of that branch and the analysis results and information for that branch will become available after the analysis is complete.
 
+!!! note
+    **If you're using Codacy Self-hosted** you can also configure Codacy to automatically enable all new branches that are pushed to the repository.
+
```

Comment:
If it's self-hosted we will show the other version below right? Or the ideia is just to add self-hosted version bellow?

Reply:
I was taking a shortcut :sweat_smile: and trying to solve two issues on the same pull request, but I guess it's better to have separate pull requests:

- I opened the dedicated pull request https://github.com/codacy/docs/pull/1486 to add the missing Self-hosted option **Auto-enable new branches**, and I'm reverting those changes on this pull request
- The scope of this pull request is to refactor the docs to take into account the migrated **Branches** tab, while keeping the old content for Self-hosted

I'm hoping that this makes everything clearer. :pray: